pages/common/md5sum.md

```diff
@@ -13,3 +13,7 @@
 - Read a file of MD5SUMs and verify all files have matching checksums:
 
 `md5sum -c {{filename.md5}}`
+
+- Calculate MD5 checksum from input text:
+
+`echo "{{your_text}}" | md5sum`
```

Comment:
```suggestion
`echo "{{message}}" | md5sum`
```

Reply:
We're not sending any "message" anywhere - it's just text, "`your_text`" - but I can change this if it's a specific style guideline

Reply:
we  use `{{message}}` in a lot of examples, also `Hello World!` but I prefer message. We try to avoid `you`, `your`, `they`, etc. Maybe this is better:
```suggestion
`echo "{{text}}" | md5sum`
```